app/src/main/java/org/oppia/app/utility/PackageManagerHelper.kt

```diff
@@ -0,0 +1,20 @@
+package org.oppia.app.utility
+
+import android.content.Context
+import android.content.pm.PackageManager
+import javax.inject.Inject
+
+/** Extension functions for PackageManager. */
```

Comment:
Not actually documenting any elements, so suggest changing to non-doc comment.

Reply:
ok